inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/service/TopicBackendWorker.java

```diff
@@ -67,9 +71,10 @@ public class TopicBackendWorker implements DisposableBean, Runnable  {
 
     TopicBackendWorker() {
         Thread thread = new Thread(this);
-        // daemon thread
-        thread.setDaemon(true);
-        thread.start();
+        this.workerExecutor = Executors
+                .newSingleThreadScheduledExecutor(
```

Comment:
Introducing a package just to use a ThreadFactory is not a good solution.

It is recommended to use the `com.google.common.util.concurrent.ThreadFactoryBuilder` class from the guava package to create a ThreadFactory.